src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Runtime/CSharpTestForEmptyStringsUsingStringLength.Fixer.cs

```diff
@@ -37,5 +37,10 @@ protected override SyntaxNode GetRightOperand(SyntaxNode binaryExpressionSyntax)
         {
             return ((BinaryExpressionSyntax)binaryExpressionSyntax).Right;
         }
+
+        protected override bool ContainsEmptyStringLiteral(SyntaxNode node)
+        {
+            return node is LiteralExpressionSyntax literalExpressionSyntax && literalExpressionSyntax.Token.ValueText.Length == 0;
+        }
```

Comment:
[cosmetic] Might be worth using the arrowed body here instead.

```suggestion
        => node is LiteralExpressionSyntax literalExpressionSyntax &&
        	literalExpressionSyntax.Token.ValueText.Length == 0;
```